src/qt/utilitydialog.cpp

```diff
@@ -58,7 +59,8 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
         ui->helpMessage->setVisible(false);
     } else {
         setWindowTitle(tr("Command-line options"));
-        QString header = "Usage:  bitcoin-qt [command-line options]                     \n";
+        const QString uri_info = wallet_enabled ? QStringLiteral(" [URI]\n\nURI must follow BIP 21") : QString();
```

Comment:
The URI has to be enclosed in single/double quotation marks. Maybe this should be added to avoid people wondering why they get to an exit message in their terminal.
```suggestion
        const QString uri_info = wallet_enabled ? QStringLiteral(" ["URI"]\n\nURI must follow BIP 21") : QString();
```